### ISSUE TITLE
Missing icons (file, product, even minimize and exit icons)

### ISSUE THREAD

**Issue by ZhangTianrong** [microsoft/vscode#126868]
Does this issue occur when all extensions are disabled?: Yes (fresh install, no extension at all)

- VS Code Version: 1.57.1
- OS Version: Windows_NT x64 10.0.19042

Steps to Reproduce:

Just do a fresh install. It happens to both my computers in the office and at home.

![image](https://user-images.githubusercontent.com/20651728/122853385-b7cf8700-d344-11eb-907d-52238adf7a06.png)

If I install another product icon pack and switch between them (no matter if I change from default to the other icon pack or from the other to default), some icons are restored (sometimes all of them but not always), and the next time I launch vscode, the problems comes back. I also see that some icon packs worked well when I first install it and the next time I open vscode, it turns into some garbled characters.

**Comment by misolori** [comment]
This looks similar to https://github.com/microsoft/vscode/issues/110288, https://github.com/microsoft/vscode/issues/85872, https://github.com/microsoft/vscode/issues/122021

@ZhangTianrong do you have any font tools installed like Mac Font Type?

**Comment by ZhangTianrong** [comment]
@misolori Thanks for pointing that out. I tried to search for existing issues with keywords like missing icons but there are just too many of them. I installed the `codicon.ttf` and it seems to solved the issue. (It seems that it doesn't take effect automatically even if you reload the window. You will have to switch between the installed product icons and then the next time a window is opened, the icons are still  there).
